tests/foundry/foundry.k.check.expected

```diff
@@ -8,34 +8,39 @@ module TOKEN-BIN-RUNTIME
     
     syntax TokenContract ::= "Token" [klabel(contract_Token)]
     
-    syntax TokenStorage ::= "x" 
+    rule  ( #binRuntime ( Token ) => #parseByteStack ( "0x608060405234801561001057600080fd5b506004361061002b5760003560e01c8063a9059cbb14610030575b600080fd5b61004361003e3660046100c9565b610045565b005b610050338383610054565b5050565b6001600160a01b038316600090815260016020526040902054610078908290610117565b6001600160a01b0380851660009081526001602052604080822093909355908416815220546100a890829061012e565b6001600160a01b039092166000908152600160205260409020919091555050565b600080604083850312156100dc57600080fd5b82356001600160a01b03811681146100f357600080fd5b946020939093013593505050565b634e487b7160e01b600052601160045260246000fd5b60008282101561012957610129610101565b500390565b6000821982111561014157610141610101565b50019056fea2646970667358221220df67d0298f49650fad9e5b2dfa3912ecd4de3790831fd79f0c186e563af14e1464736f6c634300080f0033" ) )
+      
+    
+    syntax Int ::= TokenContract "." TokenField [macro(), klabel(field_Token)]
+    
+    syntax TokenField ::= "x" [klabel(field_Token_x)]
     
-    syntax TokenStorage ::= "balances" "[" Int "]" 
+    syntax TokenField ::= "balances" "[" Int "]" [klabel(field_Token_balances)]
     
-    syntax TokenStorage ::= "allowances" "[" Int "]" "[" Int "]" 
+    syntax TokenField ::= "allowances" "[" Int "]" "[" Int "]" [klabel(field_Token_allowances)]
     
-    syntax Int ::= TokenContract "." TokenStorage [klabel(storage_Token), macro()]
+    syntax TokenField ::= "name" [klabel(field_Token_name)]
     
-    rule  ( Token.x => ( 0 +Int 0 ) )
+    rule  ( Token . x => #hashedLocation ( "Solidity" , 0 , .IntList ) )
       
     
-    rule  ( Token.balances[V0] => #hashedLocation ( "Solidity" , ( 0 +Int 1 ) , V0 ) )
+    rule  ( Token . balances [ V0 ] => #hashedLocation ( "Solidity" , 1 , V0  .IntList ) )
       
     
-    rule  ( Token.allowances[V0][V1] => #hashedLocation ( "Solidity" , #hashedLocation ( "Solidity" , ( 0 +Int 2 ) , V0 ) , V1 ) )
+    rule  ( Token . allowances [ V0 ] [ V1 ] => #hashedLocation ( "Solidity" , 2 , V0  V1  .IntList ) )
```

Comment:
@RaoulSchaffranek these ones in particular should be checked, by looking at definition of `#hashedLocation` in `hashed-locations.md`, and making sure that the before and the after both unfold to the same thing.